### ISSUE TITLE
cannot correctly serialize Json typed field

### ISSUE THREAD

**Issue by braindevices** [RobertCraigie/prisma-client-py#1038]
## Bug description
if we have model like following:
```
model Test {
  id                String    @default(uuid()) @unique
  manifest          Json
}
```
It will generate prisma model:

```
class Test(bases.BaseTest):
    id: _str
    manifest: 'fields.Json'
```

The problem is, the Json type result automatically de-serialize to a python obect. 
For example: `'{"a":1}'` become a dict `{'a': 1}`

We would expect `Test.prisma(db).create(data=test.model_dump(mode='json'))` work without problem.

However, it will not work, because during serialization the manifest does not serialize back to string. It keeps the dict type instead.

I think the correct behaviour should serialize the manifest back to Json string.

## How to reproduce

1. generate the model
2. `Test.prisma().create(data=Test(id="1", manifest='{"a": 1}').model_dump(mode='json'))`

## Expected behavior

It should just work
## Prisma information

```prisma
generator client {
  provider = "prisma-client-py"
  recursive_type_depth = -1
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}
model Test {
  id                String    @default(uuid()) @unique
  manifest          Json
}
```

## Environment & setup

- OS: ubuntu
- Database: mariadb
- Python version: 3.10
- Prisma version: 5.17.0
- prisma client python    : 0.15.0

**Comment by AutoScrape123TX** [comment]
Yes absolutely. The Json deserializer is not giving back a `str` but a `dict` on a `model_dump()`.
